github3/repos/repo.py

```diff
@@ -687,14 +687,15 @@ def create_commit(self, message, tree, parents, author=None,
         return self._instance_or_null(Commit, json)
 
     @requires_auth
-    def create_deployment(self, ref, force=False, payload='',
+    def create_deployment(self, ref, required_contexts=[], payload='',
```

Comment:
I made the change in 75a264e1b29982595802f7f91969536a0d5fa50a.
Is that correct?